app/src/main/res/values/strings.xml

```diff
@@ -1298,6 +1298,8 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_bbq_fuel_wood">Wood</string>
     <string name="quest_bbq_fuel_electric">Electric</string>
     <string name="quest_bbq_fuel_charcoal">Charcoal</string>
+    <string name="quest_bbq_fuel_not_a_bbq">Not a grill - lacks a cooking surface</string>
+    <string name="quest_bbq_fuel_not_a_bbq_confirmation_title">Is this really just a fire ring instead of full grill?</string>
```

Comment:
Other answer text is too long and likely to be truncated on most phones. Also, do you "cook" on a bbq grill?

Also, "fire ring" doesn't seem to be a common word. Are you sure that it will be understand what is meant?

https://www.youtube.com/watch?v=5WyLhwYFgmk

Reply:
That's why I'd appreciate comments from native English speakers, too. 

As for the `cook(ing)`, my understanding is that it is correct term, e.g. wikipedia [Grill disambiguation page](https://en.wikipedia.org/wiki/Grill) says (among other things):

> [Barbecue grill](https://en.wikipedia.org/wiki/Barbecue_grill), a device or surface used for **cooking** food, usually fuelled by gas or charcoal, or the part of a cooker that performs this function
[Flattop grill](https://en.wikipedia.org/wiki/Flattop_grill), a **cooking** device often used in restaurants, especially diners

But perhaps there are better words to use. What is important is that BBQ/Grill does not always have a grill/grate, but **might have a solid metal plate** (thus allowing food cooked with liquids), as noted [here](https://community.openstreetmap.org/t/amenity-bbq-without-grill-grate/9220/4). 

What differentiates `amenity=bbq` from `leisure=firepit` is that BBQ always provides _**some kind**_ of ***"cooking surface"***, which I've tried to convey there. Perhaps enumerating them (_"grill, grate, plate or similar"_)  might be better solution? But then the text might get even longer :man_shrugging: 

As for [fire ring](https://en.wikipedia.org/wiki/Fire_ring), dunno. [Fire pit](https://en.wikipedia.org/wiki/Fire_pit) might be more common, but I'm afraid people might decide _"well in my case it is not a hole in the ground (pit) so this cannot be an answer either"_. Maybe _"fire enclosure"_ could be more generic and less confusing term?

Reply:
I've changed strings to:
- _"Just a fire enclosure, without cooking surface"_ 
- _"Is this really just a fire ring / fire pit?"_

to try to use positive instead of a negative, and shorten a little while trying to use different words.

Better suggestions welcome!

Reply:
"Just a fire enclosure, without cooking surface"

Too long. "It's a fire enclosure". Then, in the dialog, there can be more text. E.g. "Are you sure?" - "A fire enclosure differs from ... by ..."

Reply:
Something like this?

![small_Screenshot_20231117_051325_de westnordost streetcomplete mn debug](https://github.com/streetcomplete/StreetComplete/assets/156656/5b73e2c0-dff4-4cae-ba5f-4a9a1f4da6dc)

Reply:
Added improved wording by @FloEdelmann , hopefully is good now